src/nvim/api/ui.c

```diff
@@ -24,21 +23,13 @@ typedef struct {
 static PMap(uint64_t) *connected_uis = NULL;
 
 void remote_ui_init(void)
+    FUNC_ATTR_NOEXPORT
 {
   connected_uis = pmap_new(uint64_t)();
-  // Add handler for "attach_ui"
-  String method = cstr_as_string("ui_attach");
-  MsgpackRpcRequestHandler handler = {.fn = remote_ui_attach, .async = false};
-  msgpack_rpc_add_method_handler(method, handler);
-  method = cstr_as_string("ui_detach");
-  handler.fn = remote_ui_detach;
-  msgpack_rpc_add_method_handler(method, handler);
-  method = cstr_as_string("ui_try_resize");
-  handler.fn = remote_ui_try_resize;
-  msgpack_rpc_add_method_handler(method, handler);
 }
 
 void remote_ui_disconnect(uint64_t channel_id)
+    FUNC_ATTR_NOEXPORT
```

Comment:
So maybe just rename FUNC_ATTR_NOEXPORT to API_NOEXPORT or something. To avoid overloading the semantics of "FUNC_ATTR_".